_includes/live-demos/premiumskinsandicons-material-outline/index.js

```diff
@@ -2,8 +2,7 @@ tinymce.init({
   selector: 'textarea#premiumskinsandicons-material-outline',
   skin: 'material-outline',
   content_css: [
```

Comment:
One more that could be simplified.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb